setup/firebase-model.md

```diff
@@ -0,0 +1,29 @@
+# Firebase model
+
+###Root level collections
+#####questions: collection of Question documents

```

Comment:
You need a line of whitespace after a title/subtitle, or it doesn't register as a title. View the file to see what I mean